lib/src/int.dart

```diff
@@ -0,0 +1,29 @@
+part of dartx;
+
+extension Ordinals<T extends int> on T {
+  /// Returns an ordinal number of `String` type for any integer
+  ///
+  /// ```dart
+  /// 101.ordinal(); // 101st
+  ///
+  /// 999218.ordinal(); // 999218th
+  /// ```
+  String ordinal() {
+    final onesPlace = this % 10;
+    final tensPlace = ((this / 10).floor()) % 10;
+    if (tensPlace == 1) {
+      return '${this}th';
```

Comment:
Please add tests for 10-15 :)

Reply:
All right.. looks like I missed few 😅 Shall be sending another PR in a moment

Reply:
Especially 101st should still be part of the tests, that was great

Reply:
Done!!